pulpcore/tests/functional/content_with_coverage.py

```diff
@@ -0,0 +1,17 @@
+import atexit
```

Comment:
Kind of like what is here. If we don't ship tests this won't ship either, which seems right.